### DISCUSSION TITLE
Problems with my code in Codewars

### DISCUSSION THREAD

**Discussion by dan93dan** [codewars/codewars.com#3264] [General]
Sorry guys, I dont know I ok with this problem.
Complete the function which returns the weekday according to the input number:

1 returns "Sunday"
2 returns "Monday"
3 returns "Tuesday"
4 returns "Wednesday"
5 returns "Thursday"
6 returns "Friday"
7 returns "Saturday"
Otherwise returns "Wrong, please enter a number between 1 and 7".

I wrote the following code:
def whatday(num):
    if num == 1:
        print("Sunday")
    if num == 2:
        print("Monday")
    if num == 3:
        print("Tuesday")
    if num == 4:
        print("Wednesday")
    if num == 5:
        print("Thursday")
    if num == 6:
        print("Friday")
    if num == 7:
        print("Saturday")
    if num >= 8:
        print("Wrong, please enter a number between 1 and 7")

and, in codewars i have the following output:

<img width="542" alt="Captura de pantalla 2024-08-13 a la(s) 5 15 32 p  m" src="https://github.com/user-attachments/assets/341b5923-dea2-49a5-a4a4-b211f5e6db3a">

Who can teach me ? :)
Thanks

**Comment by hobovsky** [comment]
Github Discussions are not meant to be used to discuss problems with specific challenges. This kind of question should be asked either in kata discourse section, or in `#help-solve` channel of Codewars Discord.

You can find some hinst related to your question in [Codewars docs](https://docs.codewars.com/training/training-example#writing-a-solution).
